appholder/src/main/java/com/android/mdl/app/MainActivity.kt

```diff
@@ -107,7 +105,12 @@ class MainActivity : AppCompatActivity() {
             // TODO: maybe bail in the future if this isn't set.
         } else {
             logInfo("referrer: $mdocReferrerUri")
-            originInfos.add(OriginInfoWebsite(1, mdocReferrerUri))
+            originInfos.add(
+              OriginInfoWebsite(
+                1,
+                mdocReferrerUri
+              )
+            )
         }
```

Comment:
Seems like unintentional noise being introduced? (Couple of other places too)

Reply:
Would be nice to fix this...

Reply:
Fixed, unintentional noise from the refactor. Ran fixes over the code base which did a few other things in addition to cleaning up some of the cruft (added finals, fixed asserts etc).